src/internal/utils/tests/sort.spec.ts

```diff
@@ -379,8 +379,32 @@ describe("createSorts", () => {
                 },
             ],
         };
-        const expectedSort: AFM.SortItem[] = [SORTED_BY_M1];
-        expect(createSorts("pushpin", afm, resultSpec, emptyVisualizationProperties)).toEqual(expectedSort);
+
+        expect(createSorts("pushpin", afm, resultSpec, emptyVisualizationProperties)).toEqual([
+            {
+                attributeSortItem: {
+                    attributeIdentifier: "a2",
+                    direction: "asc",
+                },
+            },
+        ]);
+    });
+
+    it("should return emptySort if count of attributes diffirent 2 or 3", () => {
```

Comment:
`empty sort ... different ...`
How about `should return empty sort if geo Pushpin chart has no segmentBy attribute`

Reply:
This test should be moved to `helpers/tests/sorts.spec.ts`

Reply:
move to `helpers/tests/sorts.spec.ts`